apps/date-component/README.md

```diff
@@ -0,0 +1,63 @@
+# Date component example
+
+This tutorial explains the steps to create a date using the [hof-component-date](https://github.com/UKHomeOfficeForms/hof-component-date)
+
+## Getting started
+
+Create a hof app called date-component 
```

Comment:
I know we decided not to detail the universe in every tutorial and assume the user already has some experience, but I think I would prefer it if we made it clearer, something like:


-------------------------------------------------------------
### Create the app

```bash
$ hof app date-component 
```

### Edit the app

Open the `./apps/date-component/index.js` and rename the first step from `/name` to `/date-of-birth`. 
And edit the following ....

---------------------------------------------------------------
I think it's important to phrase the instructions succinctly, perhaps as numbered steps, but definitely with order and language that leads into the next instruction. Just read it back to yourself and edit it until it makes sense